homeassistant/components/husqvarna_automower/strings.json

```diff
@@ -271,6 +271,12 @@
       },
       "stay_out_zones": {
         "name": "Avoid {stay_out_zone}"
+      },
+      "my_lawn_work_area": {
+        "name": "My lawn"
+      },
+      "work_area_work_area": {
+        "name": "{work_area}"
       }
```

Comment:
This is not a translation. We should just set `_attr_name` to the value in that case.

Reply:
Done in  https://github.com/home-assistant/core/pull/126376/commits/074b86eea8eaa60a2657f3ddc0a004f8756feb37